docs/CactbotCustomization.md

```diff
@@ -0,0 +1,366 @@
+# Cactbot Customization
+
+- [Using the cactbot UI](#using-the-cactbot-ui)
+- [User Directory Overview](#user-directory-overview)
+- [Setting Your User Directory](#setting-your-user-directory)
+- [Customizing Appearance](#customizing-appearance)
+- [Overriding Raidboss Triggers](#overriding-raidboss-triggers)
+  - [Example 1: changing the output text](#example-1--changing-the-output-text)
+  - [Example 2: making provoke work for all jobs](#example-2--making-provoke-work-for-all-jobs)
+  - [Example 3: adding custom triggers](#example-3--adding-custom-triggers)
+- [Overriding Raidboss Timelines](#overriding-raidboss-timelines)
+- [Customizing Behavior](#customizing-behavior)
+- [Debugging User Files](#debugging-user-files)
+  - [Check the OverlayPlugin log for errors](#check-the-overlayplugin-log-for-errors)
+  - [Check if your file is loaded](#check-if-your-file-is-loaded)
+  - [Check if your user file has errors](#check-if-your-user-file-has-errors)
+
+## Using the cactbot UI
+
+The best way to customize cactbot is to use the cactbot configuration UI.
+This is under
+ACT -> Plugins -> OverlayPlugin.dll -> Cactbot.
+
+This has options for things like:
+
+- setting triggers to tts
+- disabling triggers
+- changing your cactbot language
+- volume settings
+- getting rid of that cheese icon
+
+It is not possible to configure everything you might want
+through the cactbot configuration UI.
+However, it is the easiest place to start with.
+Over time, more options will be added there.
+
+These options are stored in your
+`%APPDATA%\Advanced Combat Tracker\Config\RainbowMage.OverlayPlugin.config.json`
+file.
+You should not need to edit that file directly.
+
+## User Directory Overview
+
+If the cactbot UI doesn't have the option you are looking for,
+then you may need to consider user file overrides.
+At this point, you are writing JavaScript and CSS,
+and so you might need a little bit of programming savvy.
+
+The general philosophy of cactbot is that
+any user configuration should only go in files in the user directory.
+This will prevent your changes from being clobbered during future cactbot updates.
```

Comment:
```suggestion
This will prevent your changes from being overwritten during future cactbot updates.
```
nit: Just because it might not translate well.